File: Server/Models/Funcionario.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

public class Funcionario
{
    [Key]
    public int ?Id { get; set; }

    public string ?Nome { get; set; }

    public string ?Sobrenome { get; set; }

    public string ?Endereco { get; set; }

    public string ?Telefone { get; set; }

    public List<Tarefa> ?Tarefas { get; set; }
}

File: app/lib/funcionario/domain/entities/funcionario.dart
class FuncionarioEntity {
  final int? funcionarioId;
  final String nome;
  final String sobrenome;
  final String endereco;
  final String telefone;

  const FuncionarioEntity({
    this.funcionarioId,
    required this.nome,
    required this.sobrenome,
    required this.endereco,
    required this.telefone,
  });

  String get nomeCompleto => '$nome $sobrenome';
}
